StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -350,6 +350,15 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
             }
         }
     }
+
+    func testKlarnaConfirmFlows() async throws {
+        try await _testConfirm(intentKinds: [.paymentIntent, .paymentIntentWithSetupFutureUsage, .setupIntent],
```

Comment:
Can we test that the mandate appears for paymentIntentWithSetupFutureUsage, .setupIntent and not for .paymentIntent?